sdk/flow/request.go

```diff
@@ -86,3 +94,58 @@ func MarshalChan(replyc chan *grpcflow.Reply, newItem func() proto.Message) (cha
 
 	return itemc, errc
 }
+
+// splitSetters puts user variable setters on their own channel
```

Comment:
The term "Setter" is a bit abstract. I'm wondering if we we can use "UserVar" instead? I realise this is late feedback - feel free to ignore it.

Reply:
Ah yeah, good point. It started as variables.Setter, but I should have used a more explicit term when referring to it in other packages. I'll see if I can fix it quickly.